HotelBooking.Tests/HotelBookedData.cs

```diff
@@ -0,0 +1,243 @@
+﻿using HotelBooking.Domain;
+namespace HotelBooking.Tests;
+
+public class HotelBookedData
+{
+
+    public List<RoomType> TypesRoom =
+    [
+        new RoomType{ Id = 1,  TypeName = "Luxary"},
+        new RoomType{ Id = 2,  TypeName = "Economy"},
+        new RoomType{ Id = 3,  TypeName = "Comfort"}
+    ];
+
+    public List<Passport> Passports =
+    [
+        new Passport{Id = 1, Series = 6313, Number = 123421},
+        new Passport{Id = 2, Series = 1234, Number = 567890 },
```

Comment:
Странно, мне казалось, что подобные проблемы с отступами code cleanup должен потянуть
```suggestion
        new Passport { Id = 1, Series = 6313, Number = 123421 },
        new Passport { Id = 2, Series = 1234, Number = 567890 },
```